plugins/action_generate_review_assignment_records/src/generateReviewAssignments.ts

```diff
@@ -208,13 +204,9 @@ const generateNextReviewAssignments = async ({
 
 // Helper function -- concatenates two arrays, but handles case
 // when either or both are null/undefined
-const mergeSectionRestrictions = (
-  prevArray: string[] | null | undefined,
-  newArray: string[] | null | undefined
-) => {
-  if (!prevArray) return newArray
-  else if (!newArray) return prevArray
-  else return Array.from(new Set([...prevArray, ...newArray]))
+const mergeAllowedSections = (prevArray?: string[] | null, newArray?: string[] | null) => {
+  if (!prevArray || prevArray === null) return newArray
```

Comment:
Why the OR condition? If `prevArray` is `null`, then `!prevArray` returns `true`, so why do you need both?

Reply:
Ah, it seemed to be failing when it was undefined, but maybe it the source of the problem was another. I'll change it back and re-run the tests.